erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.py

```diff
@@ -19,6 +19,10 @@ def get_link_token():
 		plaid = PlaidConnector()
 		return plaid.get_link_token()
 
+	@staticmethod
```

Comment:
Why `@staticmethod`?

Reply:
It's just to be able to quickly sync up without needing an instance of Plaid Settings. I could also do a direct whitelist call to `enqueue_sync`